README.md

```diff
@@ -1,24 +1,357 @@
-# README
+# Recycle Me Api
+[![Build Status][circleci-image]][circleci-url]
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Table of contents
+- [Overview](#overview)
+- [Endpoints](#endpoints)
+- [APIs](#apis)
+- [Endpoint and Queries](#endpoint-and-queries)
+- [Setup](#setup)
+- [Schema](#schema)
+- [Technologies](#technologies)
+- [Contributors](#contributors)
```

Comment:
A TOC is [automatically generated for readmes](https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/) by GH based on Markdown headings.